OptimizelySDKCore/OptimizelySDKCore/OPTLYFeatureDecision.m

```diff
@@ -0,0 +1,28 @@
+//
+//  OPTLYFeatureDecision.m
+//  OptimizelySDKCore
+//
+//  Created by Abdur Rafay on 20/12/2017.
+//  Copyright © 2017 Optimizely. All rights reserved.
+//
+
```

Comment:
Can you change the Copyright at the start of this file to the standard Apache 2 Optimizely
copyright notice appearing in most of the SDK *.m+*.h files, using years 2017-2018 ?
My understanding is that you (Abdur Rafay) are an Optimizely "contributor".